test/stat_tracker_test.rb

```diff
@@ -169,6 +186,20 @@ def test_it_can_identify_worst_offense
     assert_equal "Houston Dynamo", @stat_tracker.worst_offense
   end
 
+  def test_it_can_find_home_and_away_game_teams
+    @stat_tracker.find_game_teams("away").each do |game_team|
+      assert_equal "away", game_team.hoa
+    end
+
+    @stat_tracker.find_game_teams("home").each do |game_team|
+      assert_equal "home", game_team.hoa
+    end
+
+    assert_equal 3, @stat_tracker.find_game_teams("away").count && @stat_tracker.find_game_teams("home").count
+    # is it better for this line to be split up into two separate assertions
+    # or for it to be on one line but past the 80char limit?
```

Comment:
@rrabinovitch Up to you - i don't know that we'll get dinged for this, it's a small detail.